src/Symfony/Component/Notifier/Bridge/Iterable/README.md

```diff
@@ -0,0 +1,21 @@
+# Iterable Notifier
+
+Provides [Iterable](https://iterable.com/) integration for Symfony Notifier.
+
+## DSN example
+
+```
+ITERABLE_DSN=iterable://API_KEY@default?campaign_id=CAMPAIGN_ID
+```
+
+where:
+
+ - `API_KEY` is your Iterable API key
+ - `CAMPAIGN_ID` (optional) is the campaign's ID where to send messages (for e.g. push notifications) - The campaign ID can be set on the `IterableOptions` as well
```

Comment:
This sentence needs to be adjusted after the decision in https://github.com/symfony/symfony/pull/41713/files#r680538202